.env.devnet-aws

```diff
@@ -0,0 +1,12 @@
+NETWORK_NAME=testnet-aws
```

Comment:
Should it be named testnet-aws?

Reply:
For this think working, you need this value in three places:

1. file name after '.env.', e.g., .env.network
2. NETWORK_NAME inside the file, e.g., NETWORK_NAME=network
3. the same name should be used [in the combo](https://github.com/neonlabsorg/neon-compatibility/blob/f705a8f64dc3faeb9d60a38b4193e7b50078ea36/.github/workflows/consecutive.yml#L13)
e.g., 
![image](https://user-images.githubusercontent.com/2947151/147070963-c1b02d1f-d3e0-4645-be78-c36d19b16a55.png)